eslint.config.js

```diff
@@ -1,29 +1,15 @@
-import eslint from '@eslint/js';
-import eslintConfigPrettier from 'eslint-config-prettier';
-import globals from 'globals';
-import tseslint from 'typescript-eslint';
-import tsParser from '@typescript-eslint/parser';
+import { config } from '@kouts/eslint-config'
 
-export default tseslint.config(
-  eslint.configs.recommended,
-  tseslint.configs.eslintRecommended,
-  ...tseslint.configs.recommended,
-  eslintConfigPrettier,
+export default [
+  ...config({
+    vue: false,
+    semi: true
+  }),
   {
-    files: ['src/*.ts'],
-    languageOptions: {
-      parser: tsParser,
-      parserOptions: {
-        project: './tsconfig.json',
-      },
-      globals: {
-        ...globals.browser,
-        ...globals.node,
-      },
-    },
+    // Custom TypeScript rules
+    name: 'skeletos/typescript',
     rules: {
-      "@typescript-eslint/consistent-type-imports": "error",
-      "@typescript-eslint/consistent-type-exports": "error"
+      '@typescript-eslint/consistent-type-definitions': 'off',
```

Comment:
I generally prefer using types over interfaces, except in cases where I want to create a "contract" that certain classes must implement. In those situations, I find interfaces to be more appropriate.

What are your thoughts on the `consistent-type-imports` and `consistent-type-exports` rules? Do you think they provide value or might they not be worth the effort?

Reply:
Actually by default `@kouts/eslint-config` has
```
'@typescript-eslint/consistent-type-imports': [
    'error',
    {
      prefer: 'type-imports',
      fixStyle: 'inline-type-imports',
    },
  ],
 ```
 so that it enforces a consistent style for importing types in TypeScript files, specifically focusing on distinguishing between importing types and importing values.